client/index.html

```diff
@@ -1,7 +1,15 @@
 <!DOCTYPE html>
```

Comment:
There is still not angular content here (inserted by @@include) - should be removed